src/clientiface.cpp

```diff
@@ -170,6 +170,9 @@ void RemoteClient::GetNextBlocks (
 	s32 new_nearest_unsent_d = -1;
 
 	const s16 full_d_max = g_settings->getS16("max_block_send_distance");
+	s16 d_opt = g_settings->getS16("block_send_optimize_distance");
+	if (d_opt < 0) d_opt = full_d_max * 2 / 3;
```

Comment:
I'd rather this calculation wasn't here, as 'd_opt' is a constant and shouldn'tbe calculated so often.
I'd actually prefer making this commit simpler by just having a fixed settable value for 'd_opt', no scaling with 'full_d_max' when negative.